components/ws-manager-api/typescript/src/client-provider.ts

```diff
@@ -51,8 +51,8 @@ export class WorkspaceManagerClientProvider implements Disposable {
      */
     public async getStartClusterSets(
         user: User,
-        workspace: Workspace,
-        instance: WorkspaceInstance,
+        workspace?: Workspace,
+        instance?: WorkspaceInstance,
```

Comment:
It's safe to make them optional here, since there's no usage of `workspace` and `instance` inside the actual function